src/dso_api/dynamic_api/middleware.py

```diff
@@ -98,6 +98,17 @@ def process_view(self, request, view_func, view_args, view_kwargs):
             method=request.method,
             request_headers=repr(request.META),
             subject=subject,
-            data=data)
+            data=data,
+        )
 
         audit_log.info(json.dumps(log))
+
+
+class QueryParamLegalityCheckMiddleware(BaseMiddleware):
```

Comment:
Ik denk dat deze is overbodig.
In de gevaal waar we geen MandatoryQuerysets hebben - maakt het niet uit of lege parameters zitten in de request.
In de gevaal met MandatoryQuerysets - doen we check if data is beschikbaar.

Reply:
middlware is verwijderd, django doet het inderdaad out of the box. Nog even afgestemd met Michiel of we niet een 400 willen teruggeven als er een query param invalid is. Keuze was om dat niet te doen